src/client/components/responsive-video/responsive-video.vue

```diff
@@ -1,17 +1,113 @@
 <template>
-  <figure>
-    <a :href="video.url" target="_blank" rel="noopener" @click="play()">
-      <img alt="" :src="video.thumbnailUrl" :width="video.width" :height="video.height" />
-    </a>
-    <figcaption v-if="video.title">{{ video.title }}</figcaption>
-  </figure>
+  <div class="responsive-video">
+    <figure>
+      <fixed-ratio class="responsive-video__canvas" :width="video.width" :height="video.height">
+        <lazy-load>
+          <img
```

Comment:
Youtube videos often end up having thick black borders on the top and bottom, because the image dimensions are not the same as the video dimensions (YouTube sucks). Can we maybe use it as a background image with `cover` style? Could be on the `a.responsive-video__button`.